src/Symfony/Component/Cache/Messenger/EarlyExpirationHandler.php

```diff
@@ -73,7 +73,8 @@ function (CacheItem $item, float $startTime) {
         $startTime = microtime(true);
         $pool = $message->findPool($this->reverseContainer);
         $callback = $message->findCallback($this->reverseContainer);
-        $value = $callback($item);
+        $save = true;
```

Comment:
shouldn't this implement the handling of this `$save` flag ?

Reply:
My guess is at this point this is not needed since it was specifically asked to recalculate the cache item asynchronously.
There could be some use cases that I miss though.

Reply:
It does look like I am doing something wrong here. But I am also guessing if handling `$save` will make things worse for someone.

Reply:
@nicolas-grekas can you give your opinion on this ?

Reply:
>My guess is at this point this is not needed since it was specifically asked to recalculate the cache item asynchronously.

Correct!